### PR TITLE
Proxy config fixup for bad merge.

### PR DESCRIPTION
**Pull request by klausw**
The previous proxy config patch erroneously applied changes to the deleted
file package_managers.py, reviving it in the process. Delete it again, and
apply its changes to linux_virtual_machine.py.

### CODE REVIEW

**Comment by klausw** [comment]
test this please

File: perfkitbenchmarker/linux_virtual_machine.py [hunk 1]
```diff
@@ -627,9 +660,6 @@ def InstallPackages(self, packages):
 
   def Install(self, package_name):
     """Installs a PerfKit package on the VM."""
-    if ((self.is_static and not self.install_packages) or
```
Comment (ehankland) [inline perfkitbenchmarker/linux_virtual_machine.py]: I don't think we want to delete these lines

File: perfkitbenchmarker/linux_virtual_machine.py [hunk 2]
```diff
@@ -153,7 +154,30 @@ def RobustRemoteCommand(self, command):
...
+  def SetupProxy(self):
+    """Sets up proxy configuration variables for the cloud environment."""
+
+    env_file = "/etc/environment"
+    commands = []
+
+    if FLAGS.http_proxy:
+        commands.append("echo 'http_proxy=%s' | sudo tee -a %s" % (
+            FLAGS.http_proxy, env_file))
+
```
Comment (ehankland) [inline perfkitbenchmarker/linux_virtual_machine.py]: nit: remove one of the lines of whitespace?

**Comment by ehankland** [comment]
2 small comments - otherwise LGTM

File: perfkitbenchmarker/linux_virtual_machine.py (hunk 1 above)
Comment (klausw) [inline perfkitbenchmarker/linux_virtual_machine.py]: Oops, not sure what happened here. Fixed.

File: perfkitbenchmarker/linux_virtual_machine.py (hunk 2 above)
Comment (klausw) [inline perfkitbenchmarker/linux_virtual_machine.py]: Standardized the whitespace.

**Comment by klausw** [comment]
test this please.

**Comment by ehankland** [comment]
test this please.

**Pull request merged**